src/devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Ssd13xx.Commands;
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ssd13xx
+{
+    public abstract class Ssd13xx : IDisposable
+    {
+        // Multiply of screen resolution plus single command byte.
+        private const int DefaultBufferSize = 48 * 96 + 1;
+        private byte[] _genericBuffer;
```

Comment:
you should be able to initialize the buffer directly here instead of constructor